src/globus_cli/commands/group/member/add.py

```diff
@@ -0,0 +1,45 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import IdentityType, ParsedIdentity, command
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.types import FIELD_LIST_T
+
+
+def _format_added_user_id(x: dict):
+    return x["add"][0]["identity_id"]
```

Comment:
When I first saw this, it reminded me of a feature we have in the output formatter to unpack results. You can pass `response_key=...` to `formatted_print`. It's then used to index into the response data before text printing:
https://github.com/globus/globus-cli/blob/8f9267e66159ed5816c3b2db1af7731c6665190f/src/globus_cli/termio/output_formatter.py#L258

Unfortunately, `response_key` only accepts a string right now, as you can see. We could make it accept a callable, and then these helpers would be replaced with this usage:
```python
ADD_USER_FIELDS: FIELD_LIST_T = [
    ("Group ID", "group_id"),
    ("Added User ID", "identity_id"),
    ("Added User Username", "username"),
]

...

formatted_print(
    ...,
    response_key=lambda x: x["add"][0]
)
```

If you're interested in doing that here, to support this command, I'd be onboard with it. You basically need to change the use of `response_key` to check `callable(response_key)` and do little else.